pkg/build/api/types.go

```diff
@@ -197,7 +197,7 @@ type DockerBuildStrategy struct {
 // STIBuildStrategy defines input parameters specific to an STI build.
 type STIBuildStrategy struct {
 	// Image is the image used to execute the build.
-	Image string `json:"image,omitempty"`
+	From kapi.ObjectReference `json:"from,omitempty"`
```

Comment:
I like From as well, but we need to retain the Image field so you can optionally directly point to an image, just like we do elsewhere.